easybuild/easyconfigs/g/GObject-Introspection/GObject-Introspection-1.60.0-GCCcore-8.2.0-Python-3.7.2.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'ConfigureMake'
+
+name = 'GObject-Introspection'
+version = '1.60.0'
```

Comment:
I think we should go for 1.60.1 since this is still the first GO-I for 2019a

Reply:
Sure (It wasn't released when i made the PR)